src/Relude_ResultT.re

```diff
@@ -37,6 +37,10 @@ module WithMonad = (M: BsAbstract.Interface.MONAD) => {
     (aToB, ResultT(mResultA)) =>
       ResultT(M.map(resultA => Result.map(aToB, resultA), mResultA));
 
+  let semiFlatMap: 'a 'b 'e. ('a => Result.t('b, 'e),  t('a, 'e)) => t('b, 'e) =
```

Comment:
Same comment as `OptionT` - is this `subFlatMap`?  I'd love to have both `sub` and `semi` here too.

https://github.com/typelevel/cats/blob/c56edd78e6accfc276bfffad15d08d309a76076e/core/src/main/scala/cats/data/EitherT.scala#L364-L365